test/unit/spec/ForgotPassword_spec.js

```diff
@@ -230,20 +233,13 @@ function (Q, _, $, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
             data: {
               'username': 'foo',
               'factorType': 'EMAIL',
-              'relayState': ''
```

Comment:
I see... now since this is undefined we don't get an empty string. It seems good, but we are again breaking a current pattern, are we good with it? This one might be trickier if someone out there is comparing for an empty string, and now we are not even returning the `relayState` key in the object...

Reply:
@santhoshbalakrishnan  can you comment on this? I think we are good.. but I will let Santhosh confirm.

Reply:
While making the POST call to authClient, it removes any undefined parameter. If relayState was undefined, it's best to leave it undefined. If that is not sent with the original request, it won't be added back in the response.